validation/modelSat_collocation.py

```diff
@@ -6,18 +6,30 @@
 
 VERSION AND LAST UPDATE:
  v1.0  05/10/2022
+ v1.1  06/30/2022
 
 PURPOSE:
  Collocation/pairing ww3 field output results with altimeters.
  Matchups of ww3 results and satellite data are generated, for the same
   points (lat/lon) and time.
  Additional information is collocated as well, such as water depth,
-  distance to the nearest coast, ocean names, forecast zones, and
-  satellite names.
+  distance to the nearest coast, ocean names, forecast zones, satellite
+  names, and cyclone information.
 
 USAGE:
```

Comment:
For USAGE, this is a general comment.  It's most common to have the syntax for calling the routine with any arguments it might have, either from the command line, or from another routine if that is the case.  Some added description about it, I believe, is always appreciated as you've included.  Most often the call syntax is prominent right at the start of the USAGE section, then some description can follow.  These are not set in stone though.  I would recommend eventually updating this and other headers to have the call syntax somewhere in the section.  Outside of this, the documentation for headers is clear, informative, and thorough, so it is not some that is needed for this PR, but suggested to include at some point.

Reply:
I understand now. So I think part of the USAGE text I wrote can be transferred to PURPOSE, and then I leave the USAGE section for the syntax as you mentioned. In some codes, I added this as an Example:, for ex in procyclmap.py
I will include these changes in the next update.

Reply:
Yes, that exactly. Those are great then.